frontend/.gherkin-lintrc

```diff
@@ -12,8 +12,8 @@
   "no-empty-file": "on",
   "no-scenario-outlines-without-examples": "on",
   "name-length": "off",
-  "allowed-tags": ["on", {"tags": ["@manual", "@regression", "@smoke", "@add-flow", "@topology", "@knative", "@guided-tour", "@gitops", "@helm", "@monitoring", "@pipelines", "@virtualization", "@web-terminal", "@vulnerability", "@knative-eventing", "@knative-camelK", "@kafka"]}],
-  "no-restricted-tags": ["on", {"tags": ["@watch", "@wip", "@debug", "@to-do"]}],
+  "allowed-tags": ["on", {"tags": ["@e2e", @codeready-workspaces, "@health-checks", "@service-mesh",  "@manual", "@regression", "@smoke", "@add-flow", "@topology", "@knative", "@guided-tour", "@gitops", "@helm", "@monitoring", "@pipelines", "@virtualization", "@web-terminal", "@vulnerability", "@knative-eventing", "@knative-camelK", "@kafka", "@to-do", "un-verified"]}],
+  "no-restricted-tags": ["on", {"tags": ["@watch", "@wip", "@debug"]}],
```

Comment:
I'm confused, this doesn't appear related to the templates we are making 🤔

Reply:
Added @un-verified tag and included other tags as well

Reply:
Well, two things then.

1. Why are you taking this opportunity to just get extra stuff into the PR? PRs should have goals and not become just a dumping ground for any changes -- I think your non-template like changes here is not ideal (I suspect you're doing it out of convenience)
2. Shouldn't it be `@un-verified`? Today you have:
>  "@to-do", "un-verified"

Reply:
Deleted the file, I can raise a seperate pr

Reply:
I'm so confused... I asked why are you doing things that have nothing to do with the PR, so you respond with "I'll undo everything"? Help me understand here what I am missing... Do you not need to do `"@to-do", "@un-verified"` part of this work to support the work we are talking about in this PR?

> `"@e2e", @codeready-workspaces, "@health-checks", "@service-mesh",`

Has nothing to do with the discussion and work we are trying to do here, or am I mistaken? 

There is no reason to over reach and clean up things that are not related (ie tech debt), but there is also no reason not to address the topic at hand.

Reply:
I understand your point, That's the reason, I removed the file and raised a separate PR to fix .gherkin-lintrc file